SimpleKeychain/A0SimpleKeychain.m

```diff
@@ -120,6 +120,27 @@ - (BOOL)hasValueForKey:(NSString *)key {
     return status == errSecSuccess;
 }
 
+- (nonnull NSArray *)keys {
+    
+    NSMutableArray *keys = [NSMutableArray array];
+    NSDictionary *query = [self queryFindAll];
+    CFArrayRef result = nil;
+    OSStatus status = SecItemCopyMatching((__bridge CFDictionaryRef)query, (CFTypeRef *)&result);
+    if (status == errSecSuccess || status == errSecItemNotFound) {
```

Comment:
Should it be proceeding if no items were found (`errSecItemNotFound`)?

Reply:
You're right, it's useless. I've changed the code.